estate/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    'name': 'Real Estate',
+    'depends':
+        ['base'],
+    'data': [
+        'security/ir.model.access.csv',
+        'views/estate_property_type_views.xml',
+        'views/estate_property_tag_views.xml',
+        'views/estate_property_offer_views.xml',
+        'views/estate_property_views.xml',
+        'views/estate_menu.xml',
+    ],
+    'installable': True,
+    'application': True,
+    'auto_install': False,
```

Comment:
Default value are not needed (in community, this would raise a warning in runbot)
```suggestion
```